examples/esm/index.mjs

```diff
@@ -78,4 +78,15 @@ import fs from 'fs';
     signal: AbortSignal.timeout(10_000), // cancel the upload after 10 seconds
   });
   console.log(JSON.stringify(uploadResult, null, 2));
+
+  /**
+   * Tops up a wallet with Credits using tokens.
+   * Default token is AR, using Winston as the unit.
+   */
+  const topUpResult = await turboAuthClient
+    .topUpWithTokens({
+      tokenAmount: 1, /// 0.000_000_000_000_001 AR
+    })
+    .catch((err) => err); // Will throw an error with a wallet that has no tokens;
```

Comment:
Not sure this is helpful here. It's complicating the type of topUpResult for the client and also not really demonstrating a good error handling pattern.

Reply:
As we create the wallet in the script, without the catch this will cause an error when using `yarn examples` where previously it doesn't error. I added the catch so the script would continue to run. I'll remove for now